articles/cosmos-db/partners-migration-cosmosdb.md

```diff
@@ -16,7 +16,7 @@ From NoSQL migration to application development, you can choose from a variety o
 
 |**Partner**  |**Capabilities & experience**  |**Supported countries/regions**  |**Contact**  |
 |---------|---------|---------|---------|
-|![ImanisData][1]   | [Data migration](https://www.imanisdata.com/wp-content/uploads/2018/02/Imanis_DS_MongoDB_Azure_FINAL.pdf) from MongoDB to Azure Cosmos DB. <br/> Imanis Data enables global enterprises to quickly and easily migrate their on-premises or cloud MongoDB databases to Azure Cosmos DB. Imanis Data provides an agentless, incremental-forever cloud migration approach for MongoDB data stores that simplifies migration and significantly reduces downtime.|  USA  |   [Website](http://www.imanisdata.com/)      |
+|![ImanisData][1]   | [Data migration](https://www.imanisdata.com/wp-content/uploads/2018/02/Imanis_DS_MongoDB_Azure_FINAL.pdf) from MongoDB to Azure Cosmos DB. <br/> Imanis Data enables global enterprises to quickly and easily migrate their on-premises or cloud MongoDB databases to Azure Cosmos DB. Imanis Data provides an agentless, incremental-forever cloud migration approach for MongoDB data stores that simplifies migration and significantly reduces downtime.|  USA  |   [Website](https://www.imanisdata.com/)      |
```

Comment:
The Imanis Data link is not current. It redirects to `https://www.cohesity.com/products/imanis-data/`